operator/src/main/java/org/bf2/operator/managers/OperandOverrideManager.java

```diff
@@ -87,6 +87,19 @@ public List<EnvVar> applyEnvironmentTo(List<EnvVar> originals) {
             return List.copyOf(originalsOrderedMap.values());
         }
     }
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class Kafka extends OperandOverride {
+        public Map<String, Object> brokerConfig = Map.of();
```

Comment:
Should this change to `private` scope? Otherwise, we don't need getter/setter for `brokerConfig`, right?

Reply:
Yes, jackson will use the getters/setters so public visibility is unnecessary.